org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/MavenProjectImporter.java

```diff
@@ -159,6 +160,7 @@ public void importToWorkspace(IProgressMonitor monitor) throws CoreException, Op
 		MavenConfigurationImpl configurationImpl = (MavenConfigurationImpl)MavenPlugin.getMavenConfiguration();
 		configurationImpl.setDownloadSources(JavaLanguageServerPlugin.getPreferencesManager().getPreferences().isMavenDownloadSources());
 		configurationImpl.setNotCoveredMojoExecutionSeverity(JavaLanguageServerPlugin.getPreferencesManager().getPreferences().getMavenNotCoveredPluginExecutionSeverity());
+		configurationImpl.setDefaultMojoExecutionAction(PluginExecutionAction.ignore);
```

Comment:
Was this meant to fix importing of https://github.com/spring-cloud/spring-cloud-sleuth ? I still see the same number of errors with & without this change.

Reply:
It was a mistake (related to https://github.com/redhat-developer/vscode-java/issues/2889). I have fixed it.